gen/__init__.py

```diff
@@ -713,13 +732,17 @@ def make_package_filename(package_id, extension):
     cc['write_files'] = []
     # Do the transform
     for item in cc_root:
-        assert item['path'].startswith('/')
+        if is_windows:
+            tmppath = item['path'].replace('\\', '/')
```

Comment:
There is a mix of slashes throughout the code. The OS itself can generally handle this OK, but some python libraries do not. This is a case where is needs to be fixed up before we continue.
There would need to be a bunch more code present handling these file differences if we normalize it fully for Windows, rather than work around it in a few choice locations.